packages/Theme/_variables.scss

```diff
@@ -0,0 +1,38 @@
+$font: 'Open Sans', sans-serif !default;
+
+$primary: #491e8c !default;
+$primary-dark: #0e061c !default;
+$secondary: #3B1873 !default;
+$primary-light: #7e61ae !default;
+$primary-very-light: #bdafd6 !default;
+$primary-very-dark: #210e41 !default;
+$dark-grey: #3c3940 !default;
+$light-grey: #f2f0f7 !default;
+$White: #ffffff !default;
+$Black: #000000 !default;
+
+$Error: #f44336 !default;
+$Success: #4bb543 !default;
+$Info: #1e88e5 !default;
```

Comment:
Could you make these variables lowercase to have some consistency

Reply:
Fixed